src/test/java/org/fifiz/training/java/basicweatherapp/BasicWeatherAppApplicationTests.java

```diff
@@ -0,0 +1,35 @@
+package org.fifiz.training.java.basicweatherapp;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.junit.Test;
+
+/**
+ * Unit test for BasicWeatherAppApplication simple App.
+ *
+ * @author bertrand
+ */
+public class BasicWeatherAppApplicationTests {
+
+	/**
+	 * Rigorous Test :-)
+	 */
+	@Test
+	public void contextLoads() {
+		assertTrue(true);
+	}
+
+	@Test
+	public void testMain() throws IOException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/bdelion/basic-weather-app/pullRequest?prid=4408921)